ci/taos/plugins-good/pr-prebuild-clang.sh

```diff
@@ -31,12 +31,12 @@ function pr-prebuild-clang(){
     echo "########################################################################################"
     echo "[MODULE] pr-prebuild-clang: Check the code formatting style with clang-format"
     # Note that you have to install up-to-date clang-format package from llvm project.
-    # The clang-format-4.0 package includes git-clang-format as well as clang-format.
+    # The clang-format-6.0 package includes git-clang-format as well as clang-format.
```

Comment:
@gichan-jang Looks good. Could you add the below annotation for maintenance? 
# The clang-format-6.0 is used in Ubuntu 18.04 by default.

Reply:
Updated.